src/query/procedure/mg_procedure_impl.cpp

```diff
@@ -1054,6 +1106,10 @@ mgp_error mgp_map_copy(mgp_map *map, mgp_memory *memory, mgp_map **result) {
 
 void mgp_map_destroy(mgp_map *map) { DeleteRawMgpObject(map); }
 
+mgp_error mgp_map_contains_deleted(mgp_map *map, int *result) {
+  return WrapExceptions([map, result] { *result = ContainsDeleted(map); });
```

Comment:
```suggestion
  return WrapExceptions([map, result] { return ContainsDeleted(map); });
```
I think you can just do that since we have WrapExceptions which writes to `*result` object

Reply:
Didn’t work, reverting suggestion